mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -7147,13 +7257,31 @@ struct ThreadwiseCopyRewritePattern
             DictionaryAttr dictAttr =
                 layeredTransformMetadata[iter].template cast<DictionaryAttr>();
             auto gemmKExtraAttr = dictAttr.get("gemmKExtra");
+            auto gemmMExtraAttr = dictAttr.get("gemmMExtra");
```

Comment:
comment above `overrideLoadStoreHack` needs to be revised now that we are obtaining `gemmKExtra`, `gemmMExtra`, `gemmNExtra`.